src/websockets/protocol.py

```diff
@@ -653,6 +653,17 @@ def closed(self) -> bool:
 
     # Private methods - no guarantees.
 
+    @property
+    def connection_closed_exc(self) -> ConnectionClosed:
```

Comment:
Since this is creating a new exception object, I would make this a method rather than a property.